### PR TITLE
Update 0x06d-Testing-Data-Storage.md

### PR DESCRIPTION
**Pull request by kryptoknight13**
Updating "Library/Preferences"

### CODE REVIEW

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -197,7 +197,37 @@ To save the console output to a text file, click the circle with a downward-poin
 
 ![Console logs](Images/Chapters/0x06b/device_console.jpg "Monitoring console logs through XCode")
 
-##### Setting up a Web Proxy
+##### Setting up a Web Proxy using BurpSuite
+
+Burp Suite is an integrated platform for performing security testing of Mobile and Web Applications. Its various tools work seamlessly together to support the entire testing process, from initial mapping and analysis of an application’s attack surface, through to finding and exploiting security vulnerabilities. It is a toolkit where Burp proxy operates as a web proxy server, and sits as a man-in-the-middle between the browser and destination web servers. It allows the interception, inspection and modification of the raw traffic passing in both directions.
```
Comment (Sjord) [inline Document/0x06b-Basic-Security-Testing.md]: > through to

I am not sure this is correct. Maybe it should be just "to".

**Comment by Sjord** [comment]
Thank you for your contributions, @kryptoknight13.

Please use dashes instead of asterisks for the lists. Otherwise it looks good to me. 👍

**Comment by kryptoknight13** [comment]
Thanks @Sjord. Updated the changes. :)

File: Document/0x06c-Reverse-Engineering-and-Tampering.md [hunk 2]
```diff
@@ -281,8 +281,36 @@ cy# [[UIApp keyWindow] recursiveDescription].toString()
...
 <!-- TODO Instantiate objects from classes -->
-<!-- TODO Hooking native functions -->
-<!-- TODO Hooking objective-C methods -->
+
+##### Hooking native functions & objective-C methods
+* Install the application to be hooked.
+* Run the application and make it sure the app is in foreground (should not be in paused state).
+* Find the PID of the app using the command: `ps ax | grep App`.
+* Hook into the running process by using the command: `cycript -p PID`.
+* Cycript interpreter will be provided, on successful hooking. You can get the instance of the application by using the Objective-C syntax `[UIApplictaion shared Application]`.
```
Comment (Sjord) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: > `[UIApplictaion shared Application]`

Should this be `[UIApplication sharedApplication]`?

File: Document/0x06c-Reverse-Engineering-and-Tampering.md [hunk 3]
```diff
@@ -281,8 +281,36 @@ cy# [[UIApp keyWindow] recursiveDescription].toString()
...
 <!-- TODO Instantiate objects from classes -->
-<!-- TODO Hooking native functions -->
-<!-- TODO Hooking objective-C methods -->
+
+##### Hooking native functions & objective-C methods
+* Install the application to be hooked.
+* Run the application and make it sure the app is in foreground (should not be in paused state).
+* Find the PID of the app using the command: `ps ax | grep App`.
+* Hook into the running process by using the command: `cycript -p PID`.
+* Cycript interpreter will be provided, on successful hooking. You can get the instance of the application by using the Objective-C syntax `[UIApplictaion sharedApplication]`.
```
Comment (Sjord) [inline Document/0x06c-Reverse-Engineering-and-Tampering.md]: Note that `UIApplictaion` is also written incorrectly here.

**Comment by sushi2k** [comment]
Thanks @kryptoknight13. I will add also a link to the portswigger documentation.

**Pull request merged**